Project2/HandoutFsCheck.fsx

```diff
@@ -0,0 +1,95 @@
+// Michael R. Hansen   07-01-2020

```

Comment:
Is this file included by accident?

Reply:
don't know how that got in there... it's not supposed to be there, i'll take that out